packages/main/src/TableHeaderCell.ts

```diff
@@ -42,10 +42,10 @@ class TableHeaderCell extends TableCellBase {
 	/**
  	 * Defines the minimum width of the column.
 	 *
-	 * If the table is in `Popin` mode, the column will move into the popin when
+	 * If the table is in `Popin` mode, the column will move into the popin
 	 * when the minimum width does not fit anymore.
```

Comment:
I am having second thoughts about this: it should be "if ", since it is a condition and not a point in time.

Reply:
Changed it up, putting the second part into the first if sentence part